tests/PropertyList.py

```diff
@@ -102,14 +102,14 @@ def testExists(self):
 
     def testGetVector(self):
         apl = dafBase.PropertyList()
-        v = (42, 2008, 1)
+        v = [42, 2008, 1]
         apl.setInt("ints", v)
-        apl.setInt("ints2", (10, 9, 8))
+        apl.setInt("ints2", [10, 9, 8])
```

Comment:
I agree with both of you on this. We should definitely support the more Pythonic behaviour of accepting sequence types before we switch master (if we choose to do so). When exactly this work is scheduled is up to the higher-ups to decide. For now I am going to merge this ticket with the current changes.